pkg/cmd/util/net.go

```diff
@@ -19,7 +19,7 @@ import (
 func TryListen(network, hostPort string) (bool, error) {
 	l, err := net.Listen(network, hostPort)
 	if err != nil {
-		glog.V(5).Infof("Failure while checking listen on %s: %v", err)
+		glog.V(3).Infof("Failure while checking listen on %s: %v", err)
```

Comment:
these errors are returned to the caller so that they may handle them in a way deemed appropriate.  Logging here at an informational level when more detail is requested seems correct to me.  Was there a specific justification for raising the level?